topics:
  input:
    safe_points_world_coord: "/segmentation/safe_points/world_frame"
    tracks_world_coord: "/search/tracks/world_coordinates"
    platform_EKF: "/estimate/ekf"

  output:
    detected_person: "/perception/detected_person"
    safe_point: "perception/safe_point"